abci/example/kvstore/kvstore.go

```diff
@@ -186,6 +196,10 @@ func (app *Application) ProcessProposal(ctx context.Context, req *types.RequestP
 			return &types.ResponseProcessProposal{Status: types.ResponseProcessProposal_REJECT}, nil
 		}
 	}
+
+	if app.generateBlobs && !bytes.Equal(req.Blob, []byte("hiBlob")) {
```

Comment:
Same comment as above on not using a variable but a hardcoded value that is copy pasted.

Reply:
Same as above.